Dockerfile

```diff
@@ -554,3 +555,7 @@ ADD patches/tensorboard/notebook.py /opt/conda/lib/python3.6/site-packages/tenso
 
 # Set backend for matplotlib
 ENV MPLBACKEND "agg"
+
+# geospatial learn course
+RUN pip install osmnx && \
```

Comment:
Could you move up these commands too? Thanks

Reply:
done!